src/celestia/celestiacore.cpp

```diff
@@ -2497,10 +2497,9 @@ bool CelestiaCore::initSimulation(const std::filesystem::path& configFileName,
             }
             else
             {
-                WarpMeshManager *manager = GetWarpMeshManager();
-                WarpMesh *mesh = manager->find(manager->getHandle(WarpMeshInfo(config->paths.warpMeshFile)));
+                auto mesh = WarpMesh::load(config->paths.warpMeshFile);
```

Comment:
need to load warp/warpMeshFile

Reply:
I can't reproduce this from a clean build on Windows or Linux. I did see something similar at one point during development due to what looked like an incremental build error (this persisted when I switched back to the master branch), but clearing out the build directory and doing a clean rebuild fixed it.

Reply:
looks fixed now